client/src/document/molecules/localized-content-note/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Doc } from "../../types";
+
+interface LocalizedNoteContent {
+  lead: string;
+  url: string;
+  linkText: string;
+}
+
+function getNote(noteContent: LocalizedNoteContent, noteType: string) {
+  return (
+    <div className={`localized-content-note notecard inline ${noteType}`}>
+      {`${noteContent.lead}`}
+      <a href={noteContent.url}>{`${noteContent.linkText}`}</a>
+    </div>
+  );
+}
+
+export function LocalizedContentNote({
+  doc,
+  locale,
+}: {
+  doc: Doc;
+  locale: string;
+}) {
+  const activeLocaleNoteContent = {
+    fr: {
+      lead:
+        "Cette page a été traduite à partir de l'anglais par la communauté.",
+      linkText:
+        "Vous pouvez également contribuer en rejoignant la communauté francophone sur MDN.",
+      url:
+        "https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize#french_fr",
+    },
+  };
+  const inactiveLocaleNoteContent = {
+    de: {
+      lead:
+        "Der Inhalt dieser Seite wurde von der Community übersetzt, jedoch wird er nicht mehr aktiv gepflegt und kann daher veraltet sein. Wenn du mithelfen möchtest, kannst du",
+      linkText:
+        "hier herausfinden wie deaktivierte Übersetzung reaktiviert werden.",
+      url:
+        "https://github.com/mdn/translated-content#promoting-an-inactive-locale-to-tier-1",
+    },
+    es: {
+      lead:
+        "Esta página fue traducida del inglés por la comunidad, pero no se mantiene activamente, por lo que puede estar desactualizada.",
+      linkText:
+        "Si desea ayudar a mantenerlo, descubra cómo activar las configuraciones regionales inactivas.",
+      url:
+        "https://github.com/mdn/translated-content#promoting-an-inactive-locale-to-tier-1",
+    },
+  };
+  const currentActiveLocaleNoteContent = activeLocaleNoteContent[locale];
+  const currentInactiveLocaleNoteContent = inactiveLocaleNoteContent[locale];
+
+  if (doc.isTranslated && doc.isActive && currentActiveLocaleNoteContent) {
```

Comment:
```suggestion
  if (doc.isActive) {
```
And here, after, I think it should be something like this instead:
```
const noteContent = isActive && (activeLocaleNoteContent[locale] || activeLocaleNoteContent['en-US']) || inactiveLocaleNoteContent[locale] || inactiveLocaleNoteContent['en-US'];
```
That way, if if the `locale` is something like `zh-TW` and we don't have a translation for it it'll fallback to something like: `This locale active and community-driven bla bla`

Reply:
Sorry for the confusing review but if you just `const noteContent = isActive && (....` you don't need an if statement at all. 
And there's no need to look at `isTranslated` because the component isn't even loaded if that's not the case.